docs/In Depth Guides/2-web-hooks.md

```diff
@@ -219,7 +219,10 @@ Event | Description
  `container.transport.vessel_discharged` | Container discharged at port of discharge
  `container.transport.full_out` | Full out at port of discharge 
  `container.transport.empty_in` | Empty returned at destination
- `container.transport.rail_departed` | Rail depoarted from port of discharge
+ `container.transport.rail_departed` | Rail departed from port of discharge
+ `container.transport.rail_loaded` | Rail loaded at port of discharge
+ `container.transport.rail_unloaded` | Rail unloaded at port of discharge
+ `container.transport.rail_arrived` | Rail arrived at port of discharge
```

Comment:
As I look at the event generators I don't think there's anything restricting these events to port of discharge or final destination. (Perhaps we should?)